dataflow/src/main/java/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -1207,15 +1207,15 @@ public String toString() {
             sb.append("new " + type);
             if (!dimensions.isEmpty()) {
                 boolean needComma = false;
+                sb.append(" [");
```

Comment:
@smillst Right now, the `ArrayCreation` receivers are printed like this: `new String[1,2,3]`, which is not correct. It should be `new String[1][2][3]`. Therefore, moving the `[` and `]` inside the loop and removing the `needComma` check should do the job. What do you think of this?

Reply:
Sounds good, please make that change.

Reply:
Did it.